taglib/flac/flacfile.cpp

```diff
@@ -515,10 +515,26 @@ void FLAC::File::scan()
     //    ..
     // <24> Length of metadata to follow
 
-    const char blockType = header[0] & ~LastBlockFlag;
-    const bool isLastBlock = (header[0] & LastBlockFlag) != 0;
+    const char blockType           = header[0] & ~LastBlockFlag;
+    const bool isLastBlock         = (header[0] & LastBlockFlag) != 0;
     const unsigned int blockLength = header.toUInt(1U, 3U);
 
+#ifdef PRODUCT_BellHybrid
```

Comment:
I'm not sure if injecting product-specific flags into the third-party library is a good idea. How about passing a kind of 'configuration' via a constructor or by using the `with_xxx_option` method(aka builder pattern)? Then for Harmony, you would use a slightly different configuration, while for Pure it would be no changes. The ideal solution here would be to introduce these changes in such a form that they can be applied to every product without specific flags/parameters. We have already discussed it, haven't we?